README.md

```diff
@@ -92,8 +121,172 @@ export class MyComponent implements OnInit {
 }
 ```
 
+#### Usage of Forms services
+- the `exampleFormService` service is a generated service holding a `FormGroup` definition that corresponds
+ with the request data structure
+
+```html
+<form [formGroup]="exampleFormService.form" (ngSubmit)="sendForm()" class="full-width">
```

Comment:
Shouldn't we show the component `.ts` also?

Reply:
added